modules/tinymce/CHANGELOG.md

```diff
@@ -6,66 +6,66 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Indenting or outdenting list items inside a block element that was inside another list item would not work #TINY-7209
-- Switching between unordered and ordered lists would incorrectly alter any parent element that contained that list #TINY-8068
-- Custom elements on blank lines would be removed during serialization #TINY-4784
-- Pasting columns in tables could sometimes result in an invalid table #TINY-8040
-- Copying columns in tables could sometimes result in an invalid copy #TINY-8040
-- The URL detection used for `autolink` and smart paste didn't work if a path segment contained valid characters such as `!` and `:` #TINY-8069
-- Cutting content to the clipboard while selecting between the parent list and a nested list would not always set the list style to `none` on the parent list #TINY-8078
-- The autocompleter would not trigger at the start of nested list items #TINY-8759
-- Some option values for the `template` plugin weren't escaped properly when doing replacement lookups via a `RegExp` #TINY-7433
-- Copy events were not dispatched in readonly mode #TINY-6800
-- Using Ctrl+Shift+Home/End (Cmd+Shift+Up/Down on Mac) would not expand the selection when a `contenteditable="false"` element was at the edge of content #TINY-7795
-- `<pre>` tags were not preserved when copying and pasting content #TINY-7719
-- Preview and Insert Template dialogs display the correct content background color when using dark skins #TINY-8534
-- Fixed incorrect word breaks in menu dropdowns with scrollbar #TINY-8572
-- The `InsertLineBreak` command did not replace selected content #TINY-8458
-- Delete operations could behave incorrectly if the selection contains a `contenteditable="false"` element located at the edge of content #TINY-8729
-- `uploadImages` no longer triggers two `change` events if there is a removal of images on upload #TINY-8641
-- The selection is no longer incorrectly moved inside a comment by the `editor.selection.normalize()` API #TINY-7817
+- Toggling fullscreen mode with the `fullscreen` plugin now also fires the `ResizeEditor` event. #TINY-8701
+- Getting the editor's text content now returns newlines instead of an empty string if more than one empty paragraph exists. #TINY-8578
+- The `end_container_on_empty_block` option can now take a string of blocks, allowing the exiting of a blockquote element by pressing Return twice. #TINY-6559
+- The default value for `end_container_on_empty_block` option has been changed to `'blockquote'`. #TINY-6559
+- Custom elements are now treated as non-empty elements via the schema. #TINY-4784
+- The `autocompleter` menu element is now positioned instead of the wrapper. #TINY-6476
+- Choice menu items will now use the menuitemradio aria role to better reflect that only a single item can be active. #TINY-8602
+- Link menu and toolbar buttons now always execute the mceLink command. #TINY-8057
```

Comment:
```suggestion
- The `end_container_on_empty_block` option can now take a string of blocks, allowing the exiting of a blockquote element by pressing Enter or Return twice. #TINY-6559
- The default value for `end_container_on_empty_block` option has been changed to `'blockquote'`. #TINY-6559
- Link menu and toolbar buttons now always execute the `mceLink` command. #TINY-8057
- Toggling fullscreen mode when using the `fullscreen` plugin now also fires the `ResizeEditor` event. #TINY-8701
- Getting the editor's text content now returns newlines instead of an empty string if more than one empty paragraph exists. #TINY-8578
- Custom elements are now treated as non-empty elements by the schema. #TINY-4784
- The autocompleter's menu HTML element is now positioned instead of the wrapper. #TINY-6476
- Choice menu items will now use the `'menuitemradio'` aria role to better reflect that only a single item can be active. #TINY-8602
```